### PR TITLE
Change a lot of code for new functions of ES6

### PR DESCRIPTION
**Pull request by jsanchez91**
The button put after the input:password

I hope you like it

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense para saber los atributos posibles.
```
Comment (jesuslg123) [inline .vscode/launch.json]: Please remove comments :)

File: backup.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+eyIxIjp7IklEIjoxLCJuYW1lIjoiTXkgaWNvbnMgY29sbGVjdGlvbiIsImJvb2ttYXJrX2lkIjoiajV0bTlvcjNuaDAwMDAwMCIsImNyZWF0ZWQiOm51bGwsInVwZGF0ZWQiOjE1MzkyNTc4NzksImFjdGl2ZSI6MSwic291cmNlIjoibG9jYWwiLCJvcmRlciI6MCwiY29sb3IiOiIwMDAwMDAiLCJzdGF0dXMiOjF9LCJqNXRtOW9yM25oMDAwMDAwIjpbeyJpZCI6MTgxNTMzLCJ0ZWFtIjowLCJuYW1lIjoibG9jayIsImNvbG9yIjoiIzAwMDAwMCIsInByZW1pdW0iOjAsInNvcnQiOjF9LHsiaWQiOjE4MTUzNCwidGVhbSI6MCwibmFtZSI6ImxvY2siLCJjb2xvciI6IiMwMDAwMDAiLCJwcmVtaXVtIjowLCJzb3J0IjoyfV19
```
Comment (jesuslg123) [inline backup.txt]: What its this file for? Need to be committed?

File: script.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
-console.log("hi!")
```
Comment (jesuslg123) [inline script.js:1]: Thx :)

**Review by jesuslg123 — CHANGES_REQUESTED** [review]
Nice job! Please just check the comments before merge.

**Comment by jsanchez91** [comment]
It's time to accept my changes :wink:

File: script.js [hunk 4]
```diff
@@ -1,45 +1,75 @@
...
+}
+	
+function createButton(element) {
 	var button = document.createElement("Button")
-	button.textContent = "S/H"
-	button.type = "button"
-	button.onclick = function(e) {
+	var element_style = window.getComputedStyle(element)
+	IMG_LOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/lock.png')+'">'
+    IMG_UNLOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/unlock.png')+'">'
```
Comment (jesuslg123) [inline script.js]: Indentation went crazy here xD

Comment (jsanchez91) [reply script.js]: Solved

**Comment by jsanchez91** [comment]
Solved the problem

File: script.js [hunk 5]
```diff
@@ -1,45 +1,75 @@
...
 
 function initialize() {
-	var inputs = document.getElementsByTagName('input')
-	for (i = 0; i < inputs.length; i ++) {
-		var input = inputs[i]
-		var type = input.getAttribute("type")
-		if (type == 'password') {
-			processInput(input)
-		}
+	var inputs = document.querySelectorAll('input[type="password"]')
```
Comment (jesuslg123) [inline script.js:4]: Nice change 💯

File: script.js [hunk 6]
```diff
@@ -1,45 +1,75 @@
...
+}
+	
+function createButton(element) {
 	var button = document.createElement("Button")
-	button.textContent = "S/H"
-	button.type = "button"
-	button.onclick = function(e) {
+	var element_style = window.getComputedStyle(element)
+	IMG_LOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/lock.png')+'">'
+        IMG_UNLOCK = '<img style="height:100%" src="'+getUrlForBrowser('icons/unlock.png')+'">'
```
Comment (jesuslg123) [inline script.js:31]: Now looks deeper than necesary xD IDE is going crazy? Don't worry :P

File: script.js [hunk 7]
```diff
@@ -1,45 +1,75 @@
...
+	console.log(element_style.paddingTop)
```
Comment (jesuslg123) [inline script.js:32]: Do we need to log the padding?

**Review by jesuslg123 — APPROVED** [review]
Awesome work :)

**Pull request merged**